### DISCUSSION TITLE
Feature: Toggle for Disabling Auto Play when Viewing Channel's Videos Page

### DISCUSSION THREAD

**Discussion by ajm00** [night/betterttv#5984] [Ideas]
Hi! Not sure if this has been suggested before, searched and couldn't find it surprisingly.

A feature I personally _would absolutely appreciate_ is disabling the auto playing of both 'the most recent VOD' & 'the currently live livestream', when clicking on a channel's profile picture because I want to look through their recent broadcasts. I'll even keep the tab open to their videos and when I return, often it'll refresh and start auto playing again. I even have had an issue in the past of the video continuously unpausing itself after I paused it, hasn't happened recently though so hopefully that isn't an issue anymore.

**Q:** _Why do you care if the VOD auto plays? It's muted, just ignore it._

**A:** There are certain streamers I like to keep up with and I can't catch them live so I'm a VOD watcher. When a VOD auto plays, it marks that VOD as "watched" and when I click on it later, it'll continue from a random timestamp from that last auto play. 

You can call me picky and pedantic that's totally fine! 

If it's a simple implementation I would GREATLY appreciate it!! If it's a lot of work than don't worry about it lol. I'll live. I've wanted this as a feature on Twitch for so long but never thought to maybe submit an idea, so here I am!

Thanks!

**Comment by night** [comment]
This is already functionality supported by BetterTTV. You can turn off autoplaying the "Offline Channel Player" in the BetterTTV settings.
